src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1474,9 +1475,21 @@ private async Task ExecuteRestartSearchCommandAsync()
             await RefreshConsolidatablePackagesCountAsync();
         }
 
+
+        /// <summary>
+        /// For Apex tests
```

Comment:
Is there actual Apex test using this method?

Reply:
Yes, but, I updated Apex calls to keep just 1 method signature

Reply:
Not anymore.